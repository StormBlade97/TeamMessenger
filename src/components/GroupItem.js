import React from 'react';

const GroupItem = (props) => {
    return (
        <li>{props.name}</li>
    )
}

export default GroupItem