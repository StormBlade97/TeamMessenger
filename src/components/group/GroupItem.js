import React from 'react';

const GroupItem = (props) => {

    function onClick(e) {
        e.preventDefault()
        props.onClick(props.id)
    }

    return (
        <li><a href="#" onClick={onClick}>{props.name}</a></li>
    )
}

export default GroupItem