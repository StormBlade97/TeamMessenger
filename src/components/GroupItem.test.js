import React from 'react';
import GroupItem from './GroupItem';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

describe('Component: GroupItem', () => {

    it('renders a snapshot', () => {
        const tree = renderer.create(<GroupItem name="Cashiers"/>).toJSON();
        expect(tree).toMatchSnapshot();
    });
})