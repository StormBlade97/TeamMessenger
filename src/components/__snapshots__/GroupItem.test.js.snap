exports[`Component: GroupItem renders a snapshot 1`] = `
<li>
  Cashiers
</li>
`;
